examples/tests/ieee802154/README.md

```diff
@@ -0,0 +1,9 @@
+This is one of five  applications to test 802.15.4 packet reception
+and transmission. The five apps are:
+
+radio_ack: Sends packets, using a printf to signal whether they were
+           acknowledged. Also receives packets.
+radio_rx: Receives packets only.
+radio_rxtx: Sends and receives packets.
+radio_tx: Sends packets only.
+radio_tx_direct: Send packets fully formed by userprocess. This example sends an ACK packet. For forming headers, use the generic `ieee802154_send(..)` method. 
```

Comment:
Rename to "raw" here to match the kernel. (Plus, I think it makes more sense)

This same comment applies to everywhere the word "direct" appears.